R/utilities-population.R

```diff
@@ -90,3 +90,23 @@ exportPopulationToCSV <- function(population, filePath) {
   invisible()
 }
 
+#' Loads aging data (typically generated from PK-Sim) i
+#'
+#' @param filePath Full path containing an aging data table.
+#'
+#' @examples
+#' csvPath <- system.file("extdata", "aging_data.csv", package = "ospsuite")
+#'
+#' agingData <- loadAgingDataFromCSV(tempfile())
+#' @export
+loadAgingDataFromCSV <- function(filePath){

```

Comment:
@pchelle @abdullahhamadeh @PavelBal 
I am losing my mind here. 
I have 4 columns in the table. IndividualId is an integer, ParameterPath is string, Time and Value are doubles
Can I make sure R does not try to be too clever and does not convert my string into factor AND also interprets my number as numbers? Using stringAsFactor = FALSE converst all number as string.